cloud/cluster/local/config.go

```diff
@@ -2,14 +2,12 @@ package local
 
 import (
 	"github.com/scootdev/scoot/cloud/cluster"
-	"github.com/scootdev/scoot/cloud/cluster/memory"
 )
 
 type ClusterLocalConfig struct {
 	Type string
 }
 
-func (c *ClusterLocalConfig) Create() (cluster.Cluster, error) {
-	sub := Subscribe()
-	return memory.NewCluster(sub.InitialMembers, sub.Updates), nil
+func (c *ClusterLocalConfig) Create() (*cluster.Cluster, error) {
+	return cluster.NewCluster([]cluster.Node{}, make(chan interface{})), nil
```

Comment:
I'm still not sure about spots like this where you pass an arg (chan) that doesn't seem to get used.